### ISSUE TITLE
Sexist comment

### ISSUE THREAD

**Issue by teo1978** [BloombergMedia/whatiscode#78]
"Programming, despite the hype and the self-serving fantasies of programmers  the world over, isn’t the most intellectually demanding task imaginable.

Which leads one to the inescapable conclusion: The problem with women in technology isn’t the women."

The very premise to this reasoning seems to be that women are less smart than men.
You're saying that coding shouldn't be inaccessible to women because it's not an especially intellectually demanding task, WTF.
You're talking about sexism in the programming community and you make the most possible sexist comment ever.

Note: I'm not a woman and I'm usually skeptical towards taking any gender-related comment as sexist or discriminating, but this rang a bell even to me.

**Comment by davydog187** [comment]
Heres the whole context:

```
The average programmer is moderately diligent, capable of basic mathematics, has a working knowledge of one or more programming languages, and can communicate what he or she is doing to management and his or her peers. Given that a significant number of women work as journalists and editors, perform surgery, run companies, manage small businesses, and use spreadsheets, that a few even serve on the Supreme Court, and that we are no longer surprised to find women working as accountants, professors, statisticians, or project managers, it’s hard to imagine that they can’t write JavaScript. Programming, despite the hype and the self-serving fantasies of programmers the world over, isn’t the most intellectually demanding task imaginable.

Which leads one to the inescapable conclusion: The problem with women in technology isn’t the women.
```

**Comment by goosehub** [comment]
I see what you mean, but I didn't read it that way, and I doubt he wrote that because of a freudian slip. I hope there aren't many people concerned with this interpretation.

**Comment by alohaas** [comment]
When I first read the article, your interpretation didn't really come across. Then I was looking for some open issues to contribute to and it threw me for a loop! I was like, "Wait...what!" But after a little thought I think learning more toward my initial interpretation.

When Ford says, "Programming...isn't the most intellectually demanding task imaginable...The problem isn't...the women," he's also referencing his earlier comment, "The average programmer is moderately diligent, capable of basic mathematics, has a working knowledge of one or more programming languages, and can communicate what he or she is doing to management and his or her peers." I was reading this to mean that women, like men, do not have to go above and beyond to enter the field of programming, at least as far as education and certification goes – anyone who is reasonably capable can do it, as long as they put in the time and effort. Not every woman for man may be a Torvaldian legend, but he or she can certainly write code and make money doing it.

Sexism is both a delicate and heavy subject and I appreciate your vigilance, but as a woman who codes, I am not offended by this piece :) I was inspired and affirmed! I hope this helps to put this github issue in a different light.

**Comment by tlycken** [comment]
I still think this wording is problematic. To take a little more context than the first comment, but not the entire paragraph - read this:

> it’s hard to imagine that [women] can’t write JavaScript. Programming, despite the hype and the self-serving fantasies of programmers the world over, isn’t the most intellectually demanding task imaginable.

In other words, the fact that programming isn't too intellectually demanding is presented as a direct explanation to why women are in fact capable of doing it. We can connect these two sentences:

> it's hard to imagine that [women] can't write Javascript, *since* programming [...] isn't the most intellectually demanding task imaginable.

I agree that it's much more likely that Ford didn't intend this to be read that way - after all, most of the text is not at all sexist - which is why clarifying this sentence to avoid any misconceptions should be uncontroversial. I've suggested an edit in #94 that attempts to accomplish this.

**Comment by mason-bially** [comment]
I think the point of this sentence was along the lines of "regardless of whether you think women are more or less smart than men, the problem is obviously the culture, not the task."

I think it was trying to make the argument, regardless of your opinion on equality, that the task is well within reach of both genders.

**Comment by goosehub** [comment]
@mason-bially That is the best explanation of what the speaker was trying to communicate. He was trying to put himself and the reader in the mind of a sexist person so he can make the point that it wouldn't even matter even if these things were true. Doing so seems to have given some people a nasty impression of what he was trying to say, but I don't see the paragraph as an issue.

**Comment by tlycken** [comment]
@goosehub, @mason-bially: I fully agree with you that the author probably had no intention whatsoever to communicate sexism - which is why clarification shouldn't be controversial. There's no reason to allow for an interpretation that might offend some, when that interpretation is opposite of what the author intended.

**Comment by alex067** [comment]
It really sounds like you are taking this way out of context.

The text is obviously stating that; the lack of women in the field is not due to the gender it self, because qualifications of being a programmer is not as demanding, and anyone can pretty much do it.
